hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -125,8 +125,12 @@ def _test_lightningmodule(
         if metric_name == MetricsKey.CONF_MATRIX:
             continue
         score = metric_object(probs, bag_labels.view(batch_size,))
-        assert torch.all(score >= 0)
-        assert torch.all(score <= 1)
+        if metric_name == MetricsKey.COHENKAPPA:
+            assert torch.all(score[~score.isnan()] >= -1)
```

Comment:
I haven't investigated when this happens, it gave a NaN in the test. This could be related to a division by zero error https://stats.stackexchange.com/questions/157832/cohens-kappa-using-irr-and-kappa2-outputs-nan

Reply:
@dccastro where should we document this?